api/src/main/java/org/openmrs/api/db/hibernate/HibernateProgramWorkflowDAO.java

```diff
@@ -386,4 +387,54 @@ public ProgramWorkflow getWorkflowByUuid(String uuid) {
 		squery.setEntity("concept", concept);
 		return squery.list();
 	}
+
+	/**
+	 * @see org.openmrs.api.db.ProgramWorkflowDAO#getAllProgramAttributeTypes()
+	 */
+	@Override
+	public List<ProgramAttributeType> getAllProgramAttributeTypes() {
```

Comment:
You should add the boolean argument to this method, the one with no arguments shouldn't  exist in the DAO